### PR TITLE
some fixes to the runtime cleanup

### PR DESCRIPTION
**Pull request by Freax13**
It turns out that we need to fix the containerd state after removing the data folders for the snapshotters, otherwise containers won't start anymore.

### CODE REVIEW

File: packages/scripts.nix [hunk 1]
```diff
@@ -497,17 +508,10 @@
       busybox
       kubectl
       dasel
+      cleanup-images
```
Comment (katexochen) [inline packages/scripts.nix:511]: You can remove this from the let binding and refer to it as scripts.cleanup-images

File: .github/workflows/cleanup.yml
Comment (katexochen) [inline .github/workflows/cleanup.yml:1]: can this be done as part of the test?

Comment (Freax13) [reply .github/workflows/cleanup.yml:1]: What do you mean?

Comment (katexochen) [reply .github/workflows/cleanup.yml:1]: ah never mind, I'm not fully awake yet.

File: tools/bm-maintenance/deployment_tdx_snp.yml [hunk 2]
```diff
@@ -47,7 +47,7 @@ spec:
           serviceAccountName: cleanup-sa
           containers:
             - name: cleanup
-              image: ghcr.io/edgelesssys/contrast/cleanup-bm:v0.0.1
+              image: ghcr.io/edgelesssys/contrast/cleanup-bare-metal:v0.0.1
```
Comment (katexochen) [inline tools/bm-maintenance/deployment_tdx_snp.yml:50]: How is the image updated after push? How do we ensure the deployment is running with the latest image?

Comment (katexochen) [reply tools/bm-maintenance/deployment_tdx_snp.yml]: I think you'd have to set PullPolicy to always, otherwise it will just use the cached image.

Comment (Freax13) [reply tools/bm-maintenance/deployment_tdx_snp.yml]: If the image tag is `latest` the default is to always pull the image: https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting.

**Pull request merged**